File: cpp/bc1150.cpp
#include <iostream>

using namespace std;

int main()
{
	int valX, valZ, j , sum;
	valZ = j = sum = 0;
	
	cin >> valX;
	
	while(true)
	{
		cin >> valZ;
		if(valZ > valX) break;
	}
	
	for(int i = valX; i < valZ; i++)
	{
		sum += i;
		j++;
		
		if(sum > valZ) break;
	}
	
	cout << j << endl;
}

File: python/bc1150.py
valX = int(input())
valZ = 0

while True:
    valZ = int(input())
    if valZ > valX: break

som = 0
i = som

for c in range(valX, valZ+1):
    som += c
    i += 1
    if som > valZ: break

print(i)
